roles/gluster-centos/defaults/main.yml

```diff
@@ -1,3 +1 @@
 ---
-# when true, tendrl-api is installed from a source repository
```

Comment:
Ok, thanks. I personally would remove it because now it has no purpose and it can be recreated in the future but I don't mind having it there for now.